packages/cli/src/commands/sandbox/sandbox_event_handler_factory.ts

```diff
@@ -2,8 +2,7 @@ import { SandboxEventHandlerCreator } from './sandbox_command.js';
 import { BackendIdentifier } from '@aws-amplify/plugin-types';
 import { AmplifyError, UsageDataEmitter } from '@aws-amplify/platform-core';
 import { DeployResult } from '@aws-amplify/backend-deployer';
-import { COLOR } from '@aws-amplify/cli-core';
-import { printer } from '../../printer.js';
+import { COLOR, printer } from '@aws-amplify/cli-core';
```

Comment:
my vote is still to remove COLOR completely in favor of additional format options. But I'm also fine with that going in a follow up PR

Reply:
Yes.

Reply:
Cool. I'll remove `COLOR` in a follow up PR https://github.com/aws-amplify/amplify-backend/issues/1141